.github/workflows/e2e.yaml

```diff
@@ -26,27 +26,11 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v2.3.0
         with:
           node-version: ${{ matrix.node-version }}
           registry-url: https://registry.npmjs.org/
-
-      - name: Get yarn cache directory path
-        id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
-
-      - name: Restore cache of node modules and yarn cache
-        uses: actions/cache@v2
-        if: ${{ !env.ACT }}
-        id: nodejs-cache
-        with:
-          path: |
-            node_modules
-            */*/node_modules
-            ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-nodejs-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-nodejs-
+          cache: yarn
```

Comment:
Shouldn't we also set a node version? At least this is suggested by https://github.com/actions/setup-node/releases